test/CentOS-Linux-AppStream.repo

```diff
@@ -7,5 +7,6 @@
 name=CentOS Linux $releasever - AppStream
 mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=AppStream&infra=$infra
 #baseurl=http://mirror.centos.org/$contentdir/$releasever/AppStream/$basearch/os/
-gpgcheck=0
-enabled=1
\ No newline at end of file
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS
```

Comment:
missing new line a EOF

Reply:
Why is gpgcheck updated?

Reply:
Because until now there was no gpg check in 8